test/commands/email/send.test.js

```diff
@@ -0,0 +1,133 @@
+        .do(ctx => {
+          ctx.userConfig = new ConfigData();
+          ctx.userConfig.email.fromEmail = 'default@test.com';
+          ctx.userConfig.email.subjectLine = 'default';
+        })
+        .twilioCliEnv(Config)
+        .twilioCreateCommand(emailSend, flags)
+        .stdout()
+        .stderr()
+        .do(ctx => {
+          const fakePrompt = sinon.stub();
+          fakePrompt
+            .resolves({
+              to: toEmail,
+              from: fromEmail,
+              subjectLine: subjectLine,
+              text: bodyText
+            });
+          ctx.testCmd.inquirer.prompt = fakePrompt;
+        });
+      const noDefault = ({ flags = [], toEmail = '', subjectLine = '', fromEmail = '', bodyText = 'Hello world' } = {}) => test
+        .twilioCliEnv(Config)
+        .twilioCreateCommand(emailSend, flags)
+        .stdout()
+        .stderr()
+        .do(ctx => {
+          process.env.SENDGRID_API_KEY = 'SG.1134567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef_4';
+          var fakePrompt = sinon.stub();
+          fakePrompt
+            .resolves({
+              to: toEmail,
+              from: fromEmail,
+              subject: subjectLine,
+              text: bodyText
+            });
+          ctx.testCmd.inquirer.prompt = fakePrompt;
+        });
+
+      defaultSetup({ toEmail: 'jen@test.com, mike@test.com, tamu@test.com' })
+        .do(ctx => ctx.testCmd.run())
+        .exit(1)
+        .it('run email:send with no environmental variable for SendGrid key', ctx => {
+          expect(ctx.stderr).to.contain('Make sure you have an environmental variable called SENDGRID_API_KEY set up with your SendGrid API key');
+        });
+
+      noDefault({ toEmail: 'JonSnow, BranStark@winterfell', fromEmail: 'Ygritte@wall.com', flags: ['--subjectLine', 'Open ASAP'], bodyText: 'You know nothing Jon Snow.' })
+        .do(ctx => ctx.testCmd.run())
+        .exit(1)
+        .it('run email:send without defaults and with multiple recipients including an incorrect to email address', ctx => {
+          expect(ctx.stderr).to.contain('JonSnow is not a valid email');
+          expect(ctx.stderr).to.contain('Email could not be sent please re-run the command with valid email addresses');
+        });
+      noDefault({ toEmail: 'JonSnow@castleBlack.com', fromEmail: 'Ygritte@wall.com', subjectLine: 'Secret Message', bodyText: 'You know nothing Jon Snow.' })
+        .nock('https://api.sendgrid.com', api => {
+          api.post('/v3/mail/send').reply(200, {});
+        })
+        .do(ctx => ctx.testCmd.run())
+        .it('run email:send with filled out inquirer prompts', ctx => {
+          expect(ctx.stderr).to.contain('Your email containing the message "You know nothing Jon Snow." sent from Ygritte@wall.com to JonSnow@castleBlack.com with the subject line Secret Message has been sent!');
```

Comment:
Rather it just contain each of those things (emails, subject, etc.) than it match the exact format.